sounds/templatetags/display_sound.py

```diff
@@ -52,6 +52,8 @@ def display_sound(context, sound):
      'sound_id':     sound_id,
      'sound':        sound_obj,
      'sound_tags':   sound_tags,
+     'sound_user':   sound.user.username,
+     'license_name': sound.license.name,
```

Comment:
can we call select_related on user and license so that we don't have to make extra queries here?